neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py

```diff
@@ -350,4 +365,25 @@ def _main(dfSub, dfRef, WMLSref, allz_num, allz, all_MuseROIs_name, spareAD, spa
 
 	makeTablePKL(dfRef,dfSub,_os.path.join(out,UID+'_roisubsettable.pkl'))
 	plotWithRef(dfRef,WMLSref, dfSub, _os.path.join(out,UID+'_plot.png'), spareAD[0], spareBA[0])
-	makeFlagTablePKL(dfRef,allz,all_MuseROIs_name,_os.path.join(out,UID+'_flagtable.pkl'))
\ No newline at end of file
+	makeFlagTablePKL(dfRef,allz,all_MuseROIs_name,_os.path.join(out,UID+'_flagtable.pkl'))
+ 
+
+if __name__ == "__main__":
+    # dfRef = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_dfRef.pkl')
+    # dfSub = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_dfSub.pkl')
+    # WMLSref = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_WMLSref.pkl')
+    # allz_num = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_allz_num.pkl')
+    # allz = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_allz.pkl')
+    # all_MuseROIs_name = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/roi-quantification/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_all_MuseROIs_name.pkl')
+    
+    dfRef = pd.read_pickle('/home/diwu/Desktop/2_dfRef.pkl')
+    dfSub = pd.read_pickle('/home/diwu/Desktop/2_dfSub.pkl')
+    WMLSref = pd.read_pickle('/home/diwu/Desktop/2_WMLSref.pkl')
+    allz_num = pd.read_pickle('/home/diwu/Desktop/2_allz_num.pkl')
+    allz = pd.read_pickle('/home/diwu/Desktop/2_allz.pkl')
+    all_MuseROIs_name = pd.read_pickle('/home/diwu/Desktop/2_all_MuseROIs_name.pkl')
+    spareAD = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/spare-calculation/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_spareAD.pkl')
+    spareBA = pd.read_pickle('/home/diwu/Desktop/kaapana-data-to-check-brainviz/F1/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168/spare-calculation/2.16.840.1.114362.1.12066432.24920037488.604832115.605.168_spareBA.pkl')
+    pdf_path = '/home/diwu/Desktop/F1.pdf'
+    
+    _main(dfSub, dfRef, WMLSref, allz_num, allz, all_MuseROIs_name, spareAD, spareBA, pdf_path)
```

Comment:
1) Is this function needed after containerization?
2) Please remove all commented code.